docs/conf.py

```diff
@@ -436,20 +440,12 @@ def resolve_astropy_and_dev_reference(app, env, node, contnode):
 
 def setup(app):
     if sphinx_gallery is None:
-        msg = (
+        logger.warning(
             "The sphinx_gallery extension is not installed, so the "
             "gallery will not be built.  You will probably see "
             "additional warnings about undefined references due "
             "to this."
         )
-        try:
```

Comment:
I figured that compatibility with Sphinx<1.6 (1.5.6 is from May 2017) could be dropped. Most likely it was already de-facto dropped since none of the versions of Python we currently support were around at the time.